src/main/java/entities/SessionStorage.java

```diff
@@ -0,0 +1,38 @@
+package entities;
+
+import java.util.ArrayList;
+
+public class SessionStorage {
+
+    private final ArrayList<MonthlyStorage> monthlyData;
+    private final ArrayList<Expense> recurData;
+
+    /**
+     * Creates a new SessionStorage and assigns empty values to both its instance attributes
+     */
+    public SessionStorage(){
+        this.monthlyData = new ArrayList<>();
+        this.recurData = new ArrayList<>();
+    }
+
+    public void setMonthlyData(MonthlyStorage month){
+        this.monthlyData.add(month);
+    }
+
+    public void setRecurData(Expense recurExpense){
+        this.recurData.add(recurExpense);
+    }
+
+    public MonthlyStorage getMonthlyData(int currentMonth) throws EntityException {
```

Comment:
Could we change the parameter name to just month? Because this method should work with any month right, not just the current one